File: Backend/RoomMe/RoomMe.API/Models/ScheduleFullGetModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoomMe.API.Models
{
    public class ScheduleFullGetModel
    {
        public int Id { get; set; }
        public int HouseworkId { get; set; }
        public UserNicknameModel User { get; set; }
        public DateTime Date { get; set; }
        public HouseworkStatusModel Status { get; set; }
        public HouseworkSettingsModel Settings { get; set; }
    }
}

File: Android-app/domain/src/main/java/uj/roomme/domain/schedule/ScheduleFullGetModel.kt
package uj.roomme.domain.schedule

import uj.roomme.domain.housework.HouseworkModel
import uj.roomme.domain.housework.HouseworkSettingsModel
import uj.roomme.domain.housework.HouseworkStatusModel
import uj.roomme.domain.user.UserNicknameModel
import java.time.OffsetDateTime

data class ScheduleFullGetModel(
    val id: Int,
    val housework: HouseworkModel,
    val user: UserNicknameModel,
    val date: OffsetDateTime,
    val status: HouseworkStatusModel,
    val settings: HouseworkSettingsModel
)
